eng/Versions.props

```diff
@@ -270,6 +271,9 @@
     <xunitrunnerconsoleVersion>2.4.5</xunitrunnerconsoleVersion>
     <xunitrunnerwpfVersion>1.0.51</xunitrunnerwpfVersion>
     <xunitrunnervisualstudioVersion>2.4.5</xunitrunnervisualstudioVersion>
+    <xunitrunnerutilityVersion>$(xunitVersion)</xunitrunnerutilityVersion>
+    <xunitextensibilityexecutionVersion>$(xunitVersion)</xunitextensibilityexecutionVersion>
+    <xunitabstractionsVersion>2.0.3</xunitabstractionsVersion>
```

Comment:
Why is this version different?

Reply:
xunit.abstractions is not released for every version of xunit, so they are out of sync.